MobileSyncExplorerReactNative/.flowconfig

```diff
@@ -1,58 +1,63 @@
 [ignore]
```

Comment:
I created an app with the latest react-native cli and copied over its config files.